vendor/modules.txt

```diff
@@ -169,474 +169,473 @@ github.com/go-redis/redis/internal/pool
 github.com/mitchellh/copystructure
-# github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747
+# github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747 => github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747
 github.com/mitchellh/go-homedir
-# github.com/mitchellh/mapstructure v1.1.2
+# github.com/mitchellh/mapstructure v1.1.2 => github.com/mitchellh/mapstructure v1.1.2
 github.com/mitchellh/mapstructure
-# github.com/mitchellh/reflectwalk v1.0.1
+# github.com/mitchellh/reflectwalk v1.0.1 => github.com/mitchellh/reflectwalk v1.0.1
 github.com/mitchellh/reflectwalk
-# github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
+# github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd => github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
 github.com/modern-go/concurrent
-# github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742
+# github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742 => github.com/modern-go/reflect2 v0.0.0-20180701023420-4b7aa43c6742
 github.com/modern-go/reflect2
-# github.com/natefinch/lumberjack v2.0.0+incompatible
+# github.com/natefinch/lumberjack v2.0.0+incompatible => github.com/natefinch/lumberjack v2.0.0+incompatible
 github.com/natefinch/lumberjack
-# github.com/onsi/gomega v1.5.0
+# github.com/onsi/gomega v1.5.0 => github.com/onsi/gomega v1.5.0
 github.com/onsi/gomega
+github.com/onsi/gomega/format
 github.com/onsi/gomega/internal/assertion
 github.com/onsi/gomega/internal/asyncassertion
+github.com/onsi/gomega/internal/oraclematcher
 github.com/onsi/gomega/internal/testingtsupport
 github.com/onsi/gomega/matchers
-github.com/onsi/gomega/types
-github.com/onsi/gomega/internal/oraclematcher
-github.com/onsi/gomega/format
 github.com/onsi/gomega/matchers/support/goraph/bipartitegraph
 github.com/onsi/gomega/matchers/support/goraph/edge
 github.com/onsi/gomega/matchers/support/goraph/node
 github.com/onsi/gomega/matchers/support/goraph/util
-# github.com/open-policy-agent/opa v0.8.2
+github.com/onsi/gomega/types
+# github.com/open-policy-agent/opa v0.8.2 => github.com/open-policy-agent/opa v0.8.2
 github.com/open-policy-agent/opa/ast
-github.com/open-policy-agent/opa/rego
-github.com/open-policy-agent/opa/types
-github.com/open-policy-agent/opa/util
 github.com/open-policy-agent/opa/metrics
+github.com/open-policy-agent/opa/rego
 github.com/open-policy-agent/opa/storage
 github.com/open-policy-agent/opa/storage/inmem
 github.com/open-policy-agent/opa/topdown
 github.com/open-policy-agent/opa/topdown/builtins
-# github.com/openshift/api v0.0.0-20190322043348-8741ff068a47
+github.com/open-policy-agent/opa/types
+github.com/open-policy-agent/opa/util
+# github.com/openshift/api v0.0.0-20190322043348-8741ff068a47 => github.com/openshift/api v0.0.0-20190322043348-8741ff068a47
 github.com/openshift/api/apps/v1
-# github.com/opentracing/opentracing-go v1.0.2
```

Comment:
This doesn't seem right, why is everything changing?

Reply:
It does not change, 

```

# ensure_require_replace_directives_for_all_dependencies:
# - ensures all existing 'require' directives have an associated 'replace' directive pinning a version
# - adds explicit 'require' directives for all transitive dependencies
# - adds explicit 'replace' directives for all require directives (existing 'replace' directives take precedence)
function ensure_require_replace_directives_for_all_dependencies() {
```